packages/expo-tracking-transparency/plugin/src/withTrackingTransparency.ts

```diff
@@ -22,19 +22,18 @@ const withTrackingTransparency: ConfigPlugin<{
 export const withUserTrackingPermission: ConfigPlugin<{
   userTrackingPermission?: string | false;
 } | void> = (config, { userTrackingPermission } = {}) => {
-  if (userTrackingPermission === false) {
-    // TODO: Upgrade to optional chaining once Node 14+ is required
-    if (config && config.ios && config.ios.infoPlist) {
+  if (userTrackingPermission === false || userTrackingPermission === 'false') {
```

Comment:
the only code that could somehow pass the string "false" in would come from the user. As in, if they made their app.json include:
```
"plugins": [
      [
        "expo-tracking-transparency",
        {
          "userTrackingPermission": "false" <- string value instead of boolean
        }
      ]
    ],
```

It seems like an easy mistake to make, and one we should guard against (realistically, if the `NSUserTrackingUsageDescription` is just "false", their app will be rejected)

Reply:
i think we should validate that the param is the expected type, and we can use assert for this:

```
// Example plugin
module.exports = (config, { myBool }) => {
  assert(
    myBool === undefined || typeof myBool === "boolean",
    `Invalid type for myBool: ${typeof myBool}, must be undefined, true, or false`
  );
  return config;
};
```

Reply:
Removed the option for prop to be `false`- realized that it doesn't make sense for us to provide that option. If you use the Tracking Transparency framework, it is **required** by Apple that you include this permission string